docs/running-on-yarn.md

```diff
@@ -196,6 +196,15 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   It should be no larger than the global number of max attempts in the YARN configuration.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.submit.waitAppCompletion</code></td>
```

Comment:
Is the purpose of making this configurable to provide backward compatibility? I wonder if the new behavior should just be the default; I can't imagine someone actually going out of his/her way to set this to false explicitly. On the other hand people may rely on the existing behavior to detect when the application has finished. What do others think?